open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/OMRSMetadataCollectionBase.java

```diff
@@ -15,6 +15,7 @@
 
 import java.util.List;
 import java.util.Date;
+import java.util.function.Function;
```

Comment:
Are we using this?  Or just hit `Enter` too quickly (as I often do) in IntelliJ and got this magically? 😉